ecosystem/sep-0031.md

```diff
@@ -146,47 +146,70 @@ The response should be a JSON object like:
 
 ```json
 {
-   "receive":{
-      "USD":{
-         "enabled":true,
-         "fee_fixed":5,
-         "fee_percent":1,
-         "min_amount":0.1,
-         "max_amount":1000,
-         "sender_sep12_type": "sep31-sender",
-         "receiver_sep12_type": "sep31-receiver",
-         "fields":{
-            "transaction":{
-               "receiver_routing_number":{
-                  "description": "routing number of the destination bank account"
-               },
-               "receiver_account_number":{
-                  "description": "bank account number of the destination"
-               },
-               "type":{
-                  "description": "type of deposit to make",
-                  "choices":[
-                     "SEPA",
-                     "SWIFT"
-                  ]
-               }
-            }
-         }
+  "receive":{
+    "USD":{
+      "enabled":true,
+      "fee_fixed":5,
+      "fee_percent":1,
+      "min_amount":0.1,
+      "max_amount":1000,
+      "sender_sep12_required": true,
+      "receiver_sep12_required": true,
+      "sender_sep12_types": {
```

Comment:
💡 I think these fields would benefit from being grouped inside a sep12 object. It makes it really clear where the sep12 info begins and ends in this info.
```javascript
      "sep12": {
        "sender": {
          "required": true,
          "types": {
            // ..
          }
        },
        "receiver": {
          "required": true,
          "types": {
            // ..
          }
        }
```
I also think if types are only provided when required, you could leave the required field out, but I also think it is fine to include it and make that explicit.